tachyon/zk/air/expressions/evaluator.h

```diff
@@ -0,0 +1,100 @@
+#ifndef TACHYON_ZK_AIR_EXPRESSIONS_EVALUATOR_H_
+#define TACHYON_ZK_AIR_EXPRESSIONS_EVALUATOR_H_
+
+#include <vector>
+
+#include "tachyon/base/logging.h"
+#include "tachyon/math/matrix/matrix_types.h"
```

Comment:
Unused I believe? If this is included due to `Eigen::Index`, perhaps `#include "third_party/eigen3/Eigen/Core"` would be better suited.

Reply:
Now, evaluator takes `RowMajorMatrix<F>&` instead of `const F *`